rmgpy/statmech/torsionTest.py

```diff
@@ -32,426 +32,433 @@
+        for temperature, s_exp in zip(t_list, s_exp_list):
+            s_act = self.mode.getEntropy(temperature)
+            self.assertAlmostEqual(s_exp, s_act, delta=1e-3 * s_exp)
 
-    def test_getSumOfStates_classical_cosine(self):
+    def test_get_sum_of_states_classical_cosine(self):
         """
         Test the HinderedRotor.getSumOfStates() method using a cosine potential
         in the classical limit.
         """
         self.mode.quantum = False
         self.mode.fourier = None
-        Elist = numpy.arange(0, 10000*11.96, 1*11.96)
-        sumStates = self.mode.getSumOfStates(Elist)
-        densStates = self.mode.getDensityOfStates(Elist)
-        for n in range(10, len(Elist)):
-            self.assertTrue(0.8 < numpy.sum(densStates[0:n]) / sumStates[n-1] < 1.25, '{0} != {1}'.format(numpy.sum(densStates[0:n]), sumStates[n]))
+        e_list = np.arange(0, 10000 * 11.96, 1 * 11.96)
+        sum_states = self.mode.getSumOfStates(e_list)
+        dens_states = self.mode.getDensityOfStates(e_list)
+        for n in range(10, len(e_list)):
+            self.assertTrue(0.8 < np.sum(dens_states[0: n]) / sum_states[n - 1] < 1.25,
+                             '{0} != {1}'.format(np.sum(dens_states[0: n]), sum_states[n]))
 
-    def test_getSumOfStates_classical_fourier(self):
+    def test_get_sum_of_states_classical_fourier(self):
         """
         Test the HinderedRotor.getSumOfStates() method using a Fourier series
         potential in the classical limit.
         """
         self.mode.quantum = False
-        Elist = numpy.arange(0, 10000*11.96, 1*11.96)
+        e_list = np.arange(0, 10000 * 11.96, 1 * 11.96)
         try:
-            sumStates = self.mode.getSumOfStates(Elist)
+            sum_states = self.mode.getSumOfStates(e_list)
             self.fail('NotImplementedError not raised by HinderedRotor.getSumOfStates()')
         except NotImplementedError:
             pass
-        
-    def test_getSumOfStates_quantum_cosine(self):
+
+    def test_get_sum_of_states_quantum_cosine(self):
         """
         Test the HinderedRotor.getSumOfStates() method using a cosine potential
         in the quantum limit.
         """
         self.mode.quantum = True
         self.mode.fourier = None
-        Elist = numpy.arange(0, 10000*11.96, 1*11.96)
-        sumStates = self.mode.getSumOfStates(Elist)
-        densStates = self.mode.getDensityOfStates(Elist)
-        for n in range(10, len(Elist)):
-            self.assertTrue(0.8 < numpy.sum(densStates[0:n]) / sumStates[n-1] < 1.25, '{0} != {1}'.format(numpy.sum(densStates[0:n]), sumStates[n]))
+        e_list = np.arange(0, 10000 * 11.96, 1 * 11.96)
+        sum_states = self.mode.getSumOfStates(e_list)
+        dens_states = self.mode.getDensityOfStates(e_list)
+        for n in range(10, len(e_list)):
+            self.assertTrue(0.8 < np.sum(dens_states[0: n]) / sum_states[n-1] < 1.25,
```

Comment:
spaces around `-`

Reply:
I agree with spaces around `-`, but we should remove the space after `:` in this instance.

Reply:
now [0:n - 1]